reference/composedshader.md

```diff
@@ -0,0 +1,105 @@
+## ComposedShader
+
+```
+ComposedShader {
+  vrmlField MFNode fields []
+  vrmlField MFNode parts  []
```

Comment:
For me the main problem is to be consistent with the already existing nodes.
For the moments the VRML and Webots nodes have quite good and descriptive names. But now we are adding fields and nodes wiht names that are very generic and not descriptive.
This is not coherent inside the Webots nodes specification.